### PR TITLE
draft zoon post

### PR DESCRIPTION
**Pull request by stefaniebutland**
@sckott and @karthik 
In addition to checking the content of the post, please check on the following:
* font/formatting of the questions
* within the text there are links to "(code)" and "(free pdf)". Should these go elsewhere?
* in references, when I render locally, doi links show up above the references to papers. Not sure how to fix this.
* does the Africa map need to be smaller and if so, do I need to change size locally and add back to repo?
* check the caption under the Africa map and advise whether I should do this differently for the reference, CC BY licence or general text formatting

### CODE REVIEW

**Comment by sckott** [comment]
* I would do the doi links as `https://doi.org/<doi>` instead of `http://dx.doi.org/<doi>` 
* For the DOI links, just remove  the newline between the citation and the doi link
* i think the map size looks good
* i think map caption looks good

**Comment by stefaniebutland** [comment]
Thanks @sckott 
Monday I'll fix the doi's, will incorporate some feedback from Karthik, and I got permission from Nick to use a photo of him in the post. 

What mechanism should I use to make these changes, esp since I'll need to add the Nick image file to assets?

**Comment by sckott** [comment]
you can make changes to the files on your branch `stefaniebutland-zoonpost` which will get added here

**Comment by stefaniebutland** [comment]
@sckott I've made my edits to post and have final questions for you:

- what's your opinion on how we title posts for fellowship awards? I'd like to make this consistent.
Fellowship is to a person or org for a specific project and I like having the project in the title (helps ppl see what we're funding), but as Karthik said, having "the Zoon Project" in title might be odd cause it's an unknown project. So, project name in title? or just awardee's name?

- I need to add file `nick-golding.jpg` to `assets` folder. Need your guidance on how to do that 

- Pls check placement of Nick's pic. I couldn't quickly find info on how to mess with this in markdown

- will need to change date in multiple places. Is this something you would always do just before post, or are there some I should change myself?

File: _drafts/2016-12-08-ropensci-fellowship-zoon.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+---
+name: ropensci-fellowship-zoon
+layout: post
+title: "Announcing our first fellowship awardeded to Dr Nick Golding for \"the Zoon Project\""
```
Comment (sckott) [inline _drafts/2016-12-08-ropensci-fellowship-zoon.md]: awardeded ?

**Comment by sckott** [comment]
* i think just awardee's name
* you can put that in `assets/blog-images/<folder name>` where `<folder-name>` is the blog post slug, rpobably `2016-12-12-ropensci-fellowship-zoon`
* placement probably fine, though i'll look again when the image is in the folder
* one of us can change date before pushing up final version

**Pull request merged**